lib/v2/screens/explore_screens/invite/interactor/viewmodels/invite_event.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+/// --- EVENTS
+@immutable
+abstract class InviteEvent extends Equatable {
+  const InviteEvent();
```

Comment:
We should probably make the lint auto-format the files and put space in between lines.